sdk/core/Azure.Core/src/Shared/base64.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+
+namespace Azure.Core
+{
+    internal static class Base64Url
```

Comment:
File name should be "Base64Url" not "base64" to match the class name.

Reply:
This class already exists in Shared https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/src/Shared/Base64Url.cs